default/stringtables/ja.txt

```diff
@@ -0,0 +1,15224 @@
+
+OPTIONS_SOUND_SELECT
+リストの選択
+
+OPTIONS_GENERAL_COLORS
+一般
+
+OPTIONS_CONTROL_COLORS
+UI コントロール
+
+OPTIONS_FILL_COLOR
+埋める
+
+OPTIONS_BORDER_COLOR
+国境
+
+OPTIONS_TEXT_COLOR
+テキスト
+
+OPTIONS_DEFAULT_LINK_COLOR
+デフォルトリンク
+
+OPTIONS_ROLLOVER_LINK_COLOR
+ロールオーバー リンク
+
+OPTIONS_HIGHLIGHT_COLOR
+選択の強調表示
+
+OPTIONS_DROPLIST_ARROW_COLOR
+ドロップ リストの矢印
+
+OPTIONS_STATE_BUTTON_COLOR
+選択されたトグル ボタン
+
+OPTIONS_STAT_INCREASE_COLOR
+統計の増加
+
+OPTIONS_STAT_DECREASE_COLOR
+統計の減少
+
+OPTIONS_WINDOW_COLORS
+Windows
+
+OPTIONS_INNER_BORDER_COLOR
+内側の境界線
+
+OPTIONS_OUTER_BORDER_COLOR
+外側の境界線
+
+OPTIONS_PAGE_VIDEO
+ビデオ
+
+OPTIONS_PAGE_AUDIO
+オーディオ
+
+OPTIONS_PAGE_AUTOSAVE
+自動保存
+
+OPTIONS_PAGE_UI
+Ui
+
+OPTIONS_PAGE_OBJECTS_WINDOW
+オブジェクトリスト
+
+OPTIONS_COLUMNS
+列幅
+
+OPTIONS_PAGE_COLORS
+色
+
+OPTIONS_PAGE_DIRECTORIES
+ディレクトリ
+
+OPTIONS_PAGE_LOGS
+ログ
+
+OPTIONS_PAGE_MISC
+他
+
+OPTIONS_TECH_COLORS
+最寄
+
+OPTIONS_TEXT_AND_BORDER_COLOR
+テキストと枠線
+
+OPTIONS_KNOWN_TECH_COLORS
+既知の技術
+
+OPTIONS_RESEARCHABLE_TECH_COLORS
+研究可能な技術
+
+OPTIONS_UNRESEARCHABLE_TECH_COLORS
+研究不可能な技術
+
+OPTIONS_TECH_PROGRESS_COLORS
+研究進捗バー
+
+OPTIONS_PROGRESS_BAR_COLOR
+背景
+
+OPTIONS_PROGRESS_BACKGROUND_COLOR
+バー
+
+OPTIONS_FOLDER_SETTINGS
+リソース ファイル
+
+OPTIONS_FOLDER_SAVE
+ファイルを保存する
+
+OPTIONS_SERVER_FOLDER_SAVE
+サーバー保存ファイル
+
+OPTIONS_LANGUAGE_FILE
+言語ファイル
+
+OPTIONS_VOLUME_AND_MUSIC
+ボリュームと音楽
+
+OPTIONS_MUSIC_FILE
+音楽ファイル
+
+OPTIONS_SOUND_FILE
+サウンド ファイル
+
+OPTIONS_DUMP_EFFECTS_GROUPS_DESC
+自動生成された効果の説明
+
+OPTIONS_VERBOSE_LOGGING_DESC
+デバッグ用の詳細ログ
+
+OPTIONS_VERBOSE_SITREP_DESC
+エラーのあるシチュエーション レポート
+
+OPTIONS_SHOW_IDS_AFTER_NAMES
+名前の後の ID
+
+OPTIONS_EFFECT_ACCOUNTING
+効果会計
+
+OPTIONS_EFFECTS_THREADS_UI
+処理スレッド (ユーザー インターフェイス) の効果
+
+OPTIONS_EFFECTS_THREADS_AI
+処理スレッド (AI) に影響を与えます。
+
+OPTIONS_EFFECTS_THREADS_SERVER
+処理スレッド (サーバー) の効果
+
+OPTIONS_ADD_SAVED_DESIGNS
+ゲーム開始時に保存されたすべての船のデザインを追加
+
+OPTIONS_ADD_DEFAULT_DESIGNS
+ゲーム開始時にデフォルトの船舶デザインをすべて追加
+
+OPTIONS_USE_BINARY_SERIALIZATION
+バイナリ保存ファイルの作成
+
+OPTIONS_USE_XML_ZLIB_SERIALIZATION
+XML 保存ファイルに圧縮を使用する
+
+OPTIONS_CREATE_PERSISTENT_CONFIG
+永続構成ファイルの書き込み
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_TOOLTIP_TITLE
+永続的な構成の作成 (拡張機能)
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_TOOLTIP_DESC
+'''現在の構成設定を永続的な既定の設定として保存します。
+このような設定は、config.xml の設定よりも優先されますが、コマンド ラインは優先しません。
+既定値から変更された設定のみを格納します。
+既存の永続構成ファイルはすべて上書きされます。
+
+設定がバージョン間で常に有効であるとは限りません。
+このゲーム セッションを開いていない場合、以前に保存したウィンドウの設定が保持されない場合があります。
+
+See pedia article [[CONFIG_GUIDE_TITLE]] for more info.'''
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_SUCCESS
+persistent_config.xml が正常に作成されました (再作成されました)
+
+OPTIONS_CREATE_PERSISTENT_CONFIG_FAILURE
+persistent_config.xml を作成できません。
+
+
+##
+## Main map window
+##
+
+# %1% current turn number.
+MAP_BTN_TURN_UPDATE
+%1% を回す
+
+# %1% number of frames currently displayed per second.
+MAP_INDICATOR_FPS
+%1% FPS
+
+# %1% number of 'universe units' which the bar scale on the map represents.
+MAP_SCALE_INDICATOR
+<s>%1% uu</s>
+
+MAP_BTN_MENU
+メニュー
+
+MAP_BTN_MENU_DESC
+'''
```

Comment:
That seems very broken.

Reply:
String tables expect key - values pairs:

Single line value
```
KEY
val
```

Multiline  (value starts from the triple qoute and ends with triple quote)
```
KEY
'''
v
a
l
'''
```

Examples of invalid syntax
```
KEY
KEY
val
```

```
KEY
v'''
al'''
```

Open quotes can shadow a huge block of translation for some tools.
```
KEY
'''
KEY
```